wire-ios-sync-engine/Source/Synchronization/Strategies/TeamMembersDownloadRequestStrategy.swift

```diff
@@ -63,6 +63,7 @@ public final class TeamMembersDownloadRequestStrategy: AbstractRequestStrategy,
             let rawData = response.rawData,
             let payload = MembershipListPayload(rawData)
         else {
+            syncStatus.failCurrentSyncPhase(phase: .fetchingTeamMembers)
```

Comment:
The success case happens with a `func` on a different abstraction level, see screenshot. I would suggest to rather move both `syncStatus` calls in funcs or call directly to make it easier to read.

<img width="836" alt="Screenshot 2024-02-19 at 15 42 22" src="https://github.com/wireapp/wire-ios/assets/7196958/605e31a9-6f5c-48ab-87ea-1cbe0fda0a2b">

Reply:
ok yes, note that for the is phase failCurrentSyncPhase does nothing